### PR TITLE
feat: add a host usage indicator to "create vFolder dialog"

### PR DESCRIPTION
**Pull request by yomybaby**
- resolve https://github.com/lablup/backend.ai-webui/issues/1630
- related PR: https://github.com/lablup/backend.ai/pull/1170
- How to test: set `allowed_volume_info` to etcd.
```
# example
# the default value is the percentage.
./backend.ai mgr etcd put volumes/allowed_volume_info percentage,used-bytes,capacity-bytes
```
- TODO: add setting UI for setting allowed_volume_info

## After
Indicator : Green/ Yellow / Red  

스크린샷 2023-03-16 오후 5 58 38

스크린샷 2023-03-16 오후 5 58 24

스크린샷 2023-03-16 오후 5 57 58

### Status of selected host
스크린샷 2023-03-17 오후 1 41 54

### Info modal
image

---

### Get storage usage info according to the changed server structure.
it is under the usage object. ([ref](https://github.com/lablup/backend.ai/pull/1170/files#diff-ac5fe69367ba4b06e41987788344c84b1b28efc8fbc0d0b176b96124da60259aR699-R703))
```
f"{proxy_name}:{volume_data['name']}": {
            "backend": volume_data["backend"],
            "capabilities": volume_data["capabilities"],
            "usage": await fetch_allowed_volume_usage(
                storage_manager=root_ctx.storage_manager,
                proxy_name=proxy_name,
                volume_name=volume_data["name"],
            ),
        }
```
### Indicator: reduce font size
image

### Info modal: add left-padding
image

### CODE REVIEW

**Comment by Sujin-Kim1** [comment]
@yomybaby It needs to be processed today, so I'll proceed.

File: src/components/backend-ai-data-view.ts [hunk 1]
```diff
@@ -377,16 +411,47 @@ export default class BackendAIData extends BackendAIPage {
...
+              let percentage = this.storageProxyInfo[item] && this.storageProxyInfo[item]?.percentage;
+              return html` <mwc-list-item
+                hasMeta
+                value="${item}"
+                ?selected="${item === this.vhost}"
+              >
+                <div class="horizontal layout justified center">
+                  <span>${item}</span>
+                  ${true
+                    ? html`
```
Comment (agatha197) [inline src/components/backend-ai-data-view.ts]: I think this condition is always true. Is there a reason why you added this?

Comment (yomybaby) [reply src/components/backend-ai-data-view.ts]: Sorry, I forgot to add a comment. After https://github.com/lablup/backend.ai/pull/1170 is merged,  I will update this part as a condition of visibility.

Comment (yomybaby) [reply src/components/backend-ai-data-view.ts]: Oh! you already removed it! Thank you

**Review by lizable — APPROVED** [review]
LGTM.

**Pull request merged**